src/MainWindow.vala

```diff
@@ -483,11 +468,11 @@ namespace PantheonCalculator {
 
             entry.grab_focus ();
 
-            show_all ();
+            present ();
```

Comment:
This one don't cause warnings, but, i don't think it's a good thing to call `present()` here too.